perf/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "perf"
+version = "0.1.0"
+authors = ["Benjamin Saunders <ben.e.saunders@gmail.com>"]
+edition = "2018"
+license = "MIT OR Apache-2.0"
+publish = false
+
+[dependencies]
+anyhow = "1.0.22"
+futures = "0.3.8"
+quinn = { path = "../quinn" }
+rcgen = "0.8"
+rustls = "0.19"
+structopt = "0.3"
+tokio = { version = "1.0.1", features = ["rt-multi-thread", "macros"] }
```

Comment:
I think this should use a single-threaded runtime given that `bulk` shows very clearly that multithreaded performs worse

Reply:
I was thinking about the many-connection case, but sure, might as well play to our strengths.